src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/add_product_row.html.twig

```diff
@@ -64,7 +64,11 @@
           'data-modal-title': 'Create a new invoice'|trans({}, 'Admin.Orderscustomers.Feature'),
           'data-modal-apply': 'Create'|trans({}, 'Admin.Actions'),
           'data-modal-cancel': 'Cancel'|trans({}, 'Admin.Actions'),
-          'data-modal-body': 'Are you sure you want to create a new invoice?'|trans({}, 'Admin.Orderscustomers.Notification')
+          'data-modal-body': 'Are you sure you want to create a new invoice?'|trans({}, 'Admin.Orderscustomers.Notification'),
+          'data-modal-edit-price-title': 'Edit the price'|trans({}, 'Admin.Actions'),
```

Comment:
```suggestion
          'data-modal-edit-price-title': 'Edit the price'|trans({}, 'Admin.Orderscustomers.Feature'),
```

Reply:
If it is the modal's title and not a CTA, it's best to localize it in _Admin.Orderscustomers.Feature_.